models/intel/text-to-speech-en-0001/description/text-to-speech-en-0001.md

```diff
@@ -0,0 +1,125 @@
+# text-to-speech-en-0001 (composite)
+
+## Use Case and High-Level Description
+
+This is a speech synthesis composite model that simultaneously reconstructs
+mel-spectrogram and wave form from text. The model generates wave form from symbol sequences separated by space.
+The model is built on top of the modified ForwardTacotron and modified MelGAN frameworks.
+
+## Composite model specification
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| Source framework                              | PyTorch\* |
+
+
+## Duration prediction model specification
+
+The text-to-speech-en-0001-duration-prediction model is a ForwardTacotron-based duration predictor for symbols.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| GFlops                                        | 15.84      |
+| MParams                                       | 13.569     |
+
+### Inputs
+
+Sequence, name: `input_seq`, shape: `1, 512`, format: `B,C`
+where:
+   - B - batch size
+   - C - number of symbols in sequence
+
+Sequence, name: `input_mask`, shape: `1, 1, 512`, format: `B, D, C`
+where:
+   - B - batch size
+   - D - extra dimension for multiplication
+   - C - number of symbols in sequence
+
+Mask for input sequence, name: `input_mask`, shape: `1, 1, 512`, format: `B, D, C`
+where:
+   - B - batch size
+   - D - extra dimension for multiplication
+   - C - number of symbols in sequence
+
+Mask for relative position representation in attention, name: `pos_mask`, shape: `1, 1, 512, 512`, format: `B, D, C, C`
+where:
+   - B - batch size
+   - D - extra dimension for multiplication
+   - C - number of symbols in sequence
+
+### Outputs
+
+1. Duration for input symbols, name: `duration`, shape: `1, 512, 1`, format `B, C, H`. Contains predicted duration for each of the symbol in sequence.
+   - B - batch size
+   - C - number of symbols in sequence
+   - H - empty dimension
+2. Processed embeddings, name: `embeddings`, shape: [1, 512, 256], format [BxCxH]. Contains processed embeddings for each symbol in sequence.
```

Comment:
on this line I'm tired of correcting shape and layout formating, but It should be understandable that it should be the same for all places. Could you please align?

Reply:
Done